ingestion_server/ingestion_server/indexer.py

```diff
@@ -254,7 +279,10 @@ def replicate(self, table, dest_index, query):
                 num_docs = len(es_batch)
                 log.info('Pushing {} docs to Elasticsearch.'.format(num_docs))
                 # Bulk upload to Elasticsearch in parallel.
-                list(helpers.parallel_bulk(self.es, es_batch, chunk_size=400))
+                try:
+                    self._bulk_upload(es_batch)
+                except ValueError:
+                    log.error('Failed to index chunk.')
```

Comment:
Also not sure if the batch info goes on the log here for later reference

Reply:
We'll have detailed information from exc_info